tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
Can we have one console print that emits a message that no handler was found? So users are aware that they should specify a logger